### DISCUSSION TITLE
Reverse proxy for Apache?

### DISCUSSION THREAD

**Discussion by gerroon** [vrtmrz/obsidian-livesync#314] [Q&A] [answered]
Hi

I will be setting up the couchDB using docker example in the doc. Does anyone have a working RP for Apache that i can serve under a sub path like "https://mydomain.com/obsidiansync"

thanks

**Comment by gerroon** [comment]
Nobody?

**Reply by vrtmrz** [reply]
Thank you for your reminder!

**Comment by vrtmrz** [comment, marked as answer]
Sorry for being late!
Unfortunately, there are no sample files for Apache.
However, the following configuration would work for you.
```httpd.conf

LoadModule proxy_module modules/mod_proxy.so
LoadModule proxy_http_module modules/mod_proxy_http.so
LoadModule proxy_http2_module modules/mod_proxy_http2.so

<Location "/obsidiansync">
ProxyPass http://127.0.0.1:5984
ProxyPassReverse http://127.0.0.1:5984
</Location>
```
